CHANGELOG.md

```diff
@@ -6,21 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [1.1.2] - 2021-01-05
+
 ### Changed
 
--   Run tests in GitHub
--   Use prettier to format project
--   Show all opening hours
--   Use icon instead of tooltip for outbound link
+-   [#34](https://github.com/City-of-Helsinki/outdoors-sports-map/pull/34) Run tests in GitHub
+-   [#37](https://github.com/City-of-Helsinki/outdoors-sports-map/pull/34) Use prettier to format project
```

Comment:
`#37`'s link is pointing to `#34`.

Reply:
Good point. I'll fix this in a later PR.